ui/src/app/sensors/components/sensor-details/sensor-details.tsx

```diff
@@ -88,6 +101,14 @@ export const SensorDetails = ({match, location, history}: RouteComponentProps<an
                                     }
                                 });
                             }
+                        },
+                        {
+                            title: 'Logs',
+                            iconClassName: 'fa fa-bars',
```

Comment:
is this constent with the iconograph. for other places?

Reply:
Now it's the same.